steal.lic

```diff
@@ -16,13 +16,25 @@ class Steal
 
     @stealing_options = get_data('stealing').stealing_options.reject { |target| settings.dont_steal_list.include?(target['id'].to_i) }
     @stealing_bag = settings.stealing_bag
-    hometown_data = get_data('town')[settings.hometown]
+    stealing_hometown = settings.hometown 
+    stealing_hometown = settings.force_bin_town if settings.force_bin_town
+    hometown_data = get_data('town')[stealing_hometown]
     @bin_id = hometown_data['thief_bin'] && hometown_data['thief_bin']['id']
     @stealing_towns = settings.stealing_towns
     @bin_stolen = settings.bin_stolen
     if @bin_stolen && !@bin_id
-      DRC.message("*** Binning is not yet supported in #{settings.hometown}")
-      @bin_stolen = false
+      DRC.message("*** Binning check failed first attempt for #{stealing_hometown}. Trying again:")
+      pause 2
+      hometown_data = get_data('town')[stealing_hometown]
```

Comment:
Does this need to be reassigned?

Reply:
Yea, the _usual_ cause of the failure that requires the 2nd attempt is a failure at loading town data.  I've never been able to nail down what specifically causes the problem with the load, and why it sonly seems to impact steal / burgle but a 2nd attempt usually fixes it.

Reply:
This is unintuitive, but I get that weird things happen. Could you add a comment explaining it so someone doesn't try to refactor/optimize this out in the future?

Reply:
The switch to using the DCT.get_hometown_target_id method should take care of this as it's documented in there.